{"name": "Validating the successfully login with valid credentials -- @1.1 ", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"[id=\"dropdownUserLogged\"]\"}\n  (Session info: chrome=117.0.5938.89); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n\tGetHandleVerifier [0x00007FF6FE027892+54818]\n\t(No symbol) [0x00007FF6FDF96AC2]\n\t(No symbol) [0x00007FF6FDE4DA3B]\n\t(No symbol) [0x00007FF6FDE8E4FC]\n\t(No symbol) [0x00007FF6FDE8E67C]\n\t(No symbol) [0x00007FF6FDEC9627]\n\t(No symbol) [0x00007FF6FDEAEAEF]\n\t(No symbol) [0x00007FF6FDEC75A2]\n\t(No symbol) [0x00007FF6FDEAE883]\n\t(No symbol) [0x00007FF6FDE83691]\n\t(No symbol) [0x00007FF6FDE848D4]\n\tGetHandleVerifier [0x00007FF6FE38B992+3610402]\n\tGetHandleVerifier [0x00007FF6FE3E1860+3962352]\n\tGetHandleVerifier [0x00007FF6FE3D9D4F+3930847]\n\tGetHandleVerifier [0x00007FF6FE0C3646+693206]\n\t(No symbol) [0x00007FF6FDFA1628]\n\t(No symbol) [0x00007FF6FDF9D934]\n\t(No symbol) [0x00007FF6FDF9DA62]\n\t(No symbol) [0x00007FF6FDF8E113]\n\tBaseThreadInitThunk [0x00007FFC42A77344+20]\n\tRtlUserThreadStart [0x00007FFC439626B1+33]\n\n", "trace": "  File \"C:\\Desenvolvimento\\python\\FangFarmUiPythonBDDSelenium\\venv\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Desenvolvimento\\python\\FangFarmUiPythonBDDSelenium\\venv\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"feactures\\steps\\enter_login_credentials_steps.py\", line 39, in step_impl\n    systemUserLogged = context.reg.getValueUserLogged()\n  File \"C:\\Desenvolvimento\\python\\FangFarmUiPythonBDDSelenium\\feactures\\pageobjects\\LoginCredentialsPage.py\", line 25, in getValueUserLogged\n    return self.value(\"system_login_logged_ID\")\n  File \"C:\\Desenvolvimento\\python\\FangFarmUiPythonBDDSelenium\\feactures\\pageobjects\\BasePage.py\", line 71, in value\n    return self.driver.find_element(By.ID, configReader.readConfig(\"locators\", locator)).text\n  File \"C:\\Desenvolvimento\\python\\FangFarmUiPythonBDDSelenium\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 739, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n  File \"C:\\Desenvolvimento\\python\\FangFarmUiPythonBDDSelenium\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 345, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Desenvolvimento\\python\\FangFarmUiPythonBDDSelenium\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "Given I navigate to home page", "status": "passed", "start": 1695407009237, "stop": 1695407009965}, {"name": "And I navigate to login home", "status": "passed", "start": 1695407009966, "stop": 1695407010047}, {"name": "When I enter the login as \"joao.ger@maiffarm.com.br\"", "status": "passed", "start": 1695407010048, "stop": 1695407010145}, {"name": "Then I enter the password as \"123\"", "status": "passed", "start": 1695407010146, "stop": 1695407010215}, {"name": "And I click on the enter button", "status": "passed", "start": 1695407010215, "stop": 1695407010258}, {"name": "And The login of user should be displayed \"joao.ger@maiffarm.com.br\"", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"[id=\"dropdownUserLogged\"]\"}\n  (Session info: chrome=117.0.5938.89); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n\tGetHandleVerifier [0x00007FF6FE027892+54818]\n\t(No symbol) [0x00007FF6FDF96AC2]\n\t(No symbol) [0x00007FF6FDE4DA3B]\n\t(No symbol) [0x00007FF6FDE8E4FC]\n\t(No symbol) [0x00007FF6FDE8E67C]\n\t(No symbol) [0x00007FF6FDEC9627]\n\t(No symbol) [0x00007FF6FDEAEAEF]\n\t(No symbol) [0x00007FF6FDEC75A2]\n\t(No symbol) [0x00007FF6FDEAE883]\n\t(No symbol) [0x00007FF6FDE83691]\n\t(No symbol) [0x00007FF6FDE848D4]\n\tGetHandleVerifier [0x00007FF6FE38B992+3610402]\n\tGetHandleVerifier [0x00007FF6FE3E1860+3962352]\n\tGetHandleVerifier [0x00007FF6FE3D9D4F+3930847]\n\tGetHandleVerifier [0x00007FF6FE0C3646+693206]\n\t(No symbol) [0x00007FF6FDFA1628]\n\t(No symbol) [0x00007FF6FDF9D934]\n\t(No symbol) [0x00007FF6FDF9DA62]\n\t(No symbol) [0x00007FF6FDF8E113]\n\tBaseThreadInitThunk [0x00007FFC42A77344+20]\n\tRtlUserThreadStart [0x00007FFC439626B1+33]\n\n", "trace": "  File \"C:\\Desenvolvimento\\python\\FangFarmUiPythonBDDSelenium\\venv\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Desenvolvimento\\python\\FangFarmUiPythonBDDSelenium\\venv\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"feactures\\steps\\enter_login_credentials_steps.py\", line 39, in step_impl\n    systemUserLogged = context.reg.getValueUserLogged()\n  File \"C:\\Desenvolvimento\\python\\FangFarmUiPythonBDDSelenium\\feactures\\pageobjects\\LoginCredentialsPage.py\", line 25, in getValueUserLogged\n    return self.value(\"system_login_logged_ID\")\n  File \"C:\\Desenvolvimento\\python\\FangFarmUiPythonBDDSelenium\\feactures\\pageobjects\\BasePage.py\", line 71, in value\n    return self.driver.find_element(By.ID, configReader.readConfig(\"locators\", locator)).text\n  File \"C:\\Desenvolvimento\\python\\FangFarmUiPythonBDDSelenium\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 739, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n  File \"C:\\Desenvolvimento\\python\\FangFarmUiPythonBDDSelenium\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 345, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Desenvolvimento\\python\\FangFarmUiPythonBDDSelenium\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "attachments": [{"name": "screenshot", "source": "3112b8ce-b44d-41b6-90e0-f18bf0bdc9ab-attachment.png", "type": "image/png"}], "start": 1695407010259, "stop": 1695407015412}], "parameters": [{"name": "login", "value": "joao.ger@maiffarm.com.br"}, {"name": "password", "value": "123"}, {"name": "userLogged", "value": "joao.ger@maiffarm.com.br"}], "start": 1695407007699, "stop": 1695407017635, "uuid": "f01c2b5a-7fc6-44f6-b287-a429e432c580", "historyId": "0d64c722bad2b8eec175ee2b7bf7bc76", "fullName": "Enter Login Credentials: Validating the successfully login with valid credentials", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "loginCredentials"}, {"name": "feature", "value": "Enter Login Credentials"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}